repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -225,25 +225,15 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
   override def addDateHeader(name: String, timeSinceEpoch: Long): Unit =
     addHeader(name, DateUtils.formatDate(new Date(timeSinceEpoch)))
 
-  private def logHeaderWarning(headerNames: String, mayWill: String, notStill: String, name: String, value: String): Unit = {
-    logHeaderWarning(headerNames, mayWill, notStill, s"$name: $value")
-  }
-
-  private def logHeaderWarning(headerNames: String, mayWill: String, notStill: String, nameValue: String): Unit = {
-    headerWarningLogger.info(s"Calls to $headerNames after the response has been committed $mayWill be ignored.")
-    headerWarningLogger.warn(s"The following header $mayWill $notStill arrive at the client: $nameValue")
-  }
+  private def headerWarningMessage(method: String, header: String): String = s"Calls to $method after the response has been committed may be ignored -- the following header may not be modified: $header"
```

Comment:
Oh, legit. Take a header string, then you can decide whether or not to pass the name and value or just value. I saw this before, but didn't think about it in my suggestions. Nice.